azavea/raster-vision
# Make validation chips

## Overview

This PR fixes a typo that made it so the validation chips were the same as the training chips.

## Testing Instructions

* Run the chip command for an experiment where the training and validation scenes differ and observe the output. I did this for the Potsdam experiment in #408
